test/e2e/kubernetes/scripts/cloud-init-files-validate.sh

```diff
@@ -21,4 +22,8 @@ stat /boot/grub/grub.cfg | grep 'Access: (0400' || exit 1
 # validate grub configuration
 sudo grep "^\s*linux" /boot/grub/grub.cfg | grep 'audit=1' || exit 1
 # validate su configuration
-sudo grep "auth required pam_wheel.so use_uid" /etc/pam.d/su
\ No newline at end of file
+sudo grep "auth required pam_wheel.so use_uid" /etc/pam.d/su
+# validate umask configuration
+touch test-umask
+stat test-umask | grep 'Access: (0640' || exit 1
+rm -f test-umask
```

Comment:
this won't happen if exit is one

Reply:
Yeah, that's O.K., we can leave it around to troubleshoot